lib/rpush/client/active_model/wns/notification.rb

```diff
@@ -23,6 +23,8 @@ def self.included(base)
               validates :uri, presence: true
               validates :uri, format: { with: %r{https?://[\S]+} }
               validates :data, presence: true, unless: :skip_data_validation?
+              validates_with Rpush::Client::ActiveModel::PayloadDataSizeValidator,
+                             limit: 5120
```

Comment:
I assume we would want this: https://docs.oracle.com/html/E59511_02/api/platforms/wns/